code/tests/functional/tests/functions/README.md

```diff
@@ -0,0 +1 @@
+wip
```

Comment:
Is this needed?

Reply:
I was just being lazy, let me actually write it lol

Reply:
Done